azurelinuxagent/common/osutil/default.py

```diff
@@ -291,7 +300,7 @@ def enable_firewall(self, dst_ip, uid):
             logger.info("Successfully added Azure fabric firewall rules")
```

Comment:
I can't add a comment there, but can you modify the comment on line 293 to say - `# Otherwise, append all rules`

Reply:
Addressed